packages/icon/src/template.ts

```diff
@@ -0,0 +1,143 @@
+/**
+usage:
+cd packages/icon
+
+# for entire folder
+yarn svgr --template src/template.js -d dist/test src/glyphs
+
+# for one file while developing
+yarn svgr --template src/template.js -d dist/test src/glyphs/ActivityFeed.svg
+
+I have these tabs open atm:
+https://react-svgr.com/docs/custom-templates/
+https://github.com/gregberge/svgr/blob/master/packages/babel-plugin-transform-svg-component/src/index.js#L28
+https://babeljs.io/docs/en/babel-template#template-literal-usage
+https://astexplorer.net/
+*/
+
+/**
+ function createGlyphComponent(glyphName, Glyph) {
+  function GlyphComponent({ className, size = 16, title, fill, ...props }) {
+
+
+    return (
+      <Glyph
+        className={cx(
+          {
+            [fillStyle]: fill != null,
+          },
+          className,
+        )}
+        title={getGlyphTitle(glyphName, title)}
+        height={size}
+        width={size}
+        {...props}
+      />
+    );
+  }
+}
+**/
+interface FirstArg extends Record<string, any> {
+  template: {
+    smart: (opts: Record<string, any>) => Record<string, any>;
+  } & Record<string, any>;
+}
+
+interface SecondArg extends Record<string, any> {
```

Comment:
Taking from here: https://react-svgr.com/docs/custom-templates/#custom-component-template

```suggestion
interface SVGROptions extends Record<string, any> {
```

Reply:
Thank you so much. I hated this lol